io/src/lib.rs

```diff
@@ -0,0 +1,225 @@
+#![no_std]
+
+use codec::{Decode, Encode};
+use gstd::{prelude::*, ActorId};
+use scale_info::TypeInfo;
+
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Clone)]
+pub enum Move {
+    Rock,
+    Paper,
+    Scissors,
+    Lizard,
+    Spock,
+}
+
+impl Move {
+    pub fn new(number: char) -> Move {
+        match number {
+            '0' => Move::Rock,
+            '1' => Move::Paper,
+            '2' => Move::Scissors,
+            '3' => Move::Lizard,
+            '4' => Move::Spock,
+            _ => panic!("Unknown symbol in move"),
+        }
+    }
```

Comment:
I'd use `u8` instead of wide `char`.
```suggestion
    pub fn new(number: u8) -> Move {
        match number {
            b'0' => Move::Rock,
            b'1' => Move::Paper,
            b'2' => Move::Scissors,
            b'3' => Move::Lizard,
            b'4' => Move::Spock,
            _ => panic!("Unknown symbol in move"),
        }
    }
```